servo-tidy.toml

```diff
@@ -0,0 +1,24 @@
+[configs]
+skip-check-length = false
+skip-check-licenses = false
+
+[ignore]
+
+# Files that are ignored for all tidy and lint checks.
+files = [
+    # servo-tidy does not properly support workspaces yet.
+    "./Cargo.toml",
+    # There are many duplicated packages, which we probably need to review
+    # one by one.
+    "./Cargo.lock"
+]
+
+# Many directories are currently ignored while we tidy things up
+# gradually.
+directories = [
+   # Generated and upstream code combined with our own. Could use cleanup
+  "./target",
```

Comment:
The comment is indeed a little unclear, but this is a list of directories excluded from the tidy. We will gradually remove entries.